shared/eslint-config/index.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
     "@typescript-eslint/no-inferrable-types": "off",
     "no-constant-condition": "off",
     "@typescript-eslint/no-non-null-assertion": "off", // Because we have a custom no-restricted-syntax rule for this
+    "@typescript-eslint/no-base-to-string": "off", // Too many false positives for Yjs objects
```

Comment:
I've centrally disabled this lint rule for all of our projects (in `shared/eslint-config`). I previously also added a couple of these ignores on a case-by-case basis, but the lint rule isn't smart enough to not report these for Y objects, where this is not an issue, so it will just be annoying and not actually helpful.